includes/class-wc-admin-loader.php

```diff
@@ -163,28 +142,586 @@ public static function load_features() {
 	 * @todo The entry point for the embed needs moved to this class as well.
 	 */
 	public static function register_page_handler() {
-		$page_title = null;
-		$menu_title = null;
+		wc_admin_register_page(
+			array(
+				'id'     => 'woocommerce-dashboard', // Expected to be overridden if dashboard is enabled.
+				'parent' => 'woocommerce',
+				'title'  => null,
+				'path'   => self::APP_ENTRY_POINT,
+			)
+		);
+
+		// TODO: move this somewhere else?
```

Comment:
Yeah, I'm thinking loading these in from another file probably makes sense, just because of the sheer number of lines dedicated to connecting.